package.json

```diff
@@ -38,15 +39,15 @@
     "typescript": "^3.9.7"
   },
   "dependencies": {
-    "@apimatic/schema": "^0.4.1",
-    "@types/node": "^14.14.27",
```

Comment:
Still not sure if we need this or not. apimatic claims that we don't but let's keep an eye out on this and maybe revert this